chainermn/communicators/flat_communicator.py

```diff
@@ -26,11 +25,8 @@ def allreduce_grad(self, model):
         _memory_utility.pack_params(
             params, 'grad', self.gpu_buffer_a, allreduce_grad_dtype)
 
-        self.mpi_comm.Allreduce(
-            [self.gpu_buffer_a.buffer(n_bytes_total), mpi4py.MPI.FLOAT],
-            [self.gpu_buffer_b.buffer(n_bytes_total), mpi4py.MPI.FLOAT])
-        arr = self.gpu_buffer_b.array(n_elems_total)
-        arr *= (1.0 / self.size)
+        self.multinode_mean(self.gpu_buffer_a.array(n_elems_total),
```

Comment:
How about `multi_node_mean`?

There is already `create_multi_node_optimizer`

Reply:
Fixed